src/row.zig

```diff
@@ -209,6 +209,9 @@ pub const LazyList = struct {
 };
 
 inline fn _isNull(validity: [*c]u64, index: usize) bool {
+    if (validity == 0) {
```

Comment:
Do we know under which conditions this happen, since the docs only say it *might*? Would be nice to add a test for this if we can reproduce the behavior.